src/opdsrequestmanager.cpp

```diff
@@ -23,20 +23,10 @@ void OpdsRequestManager::doUpdate(const QString& currentLanguage, const QString&
     query.addQueryItem("count", QString::number(-1));
 
     // Add filter by category (if necessary)
-    if (categoryFilter != "all" && categoryFilter != "other") {
-        query.addQueryItem("tag", "_category:"+categoryFilter);
+    if (categoryFilter != "all") {
+        query.addQueryItem("category", categoryFilter);
     }
 
-    // Add "special negative" filter for "other" category (if necessary)
-    if (categoryFilter == "other") {
-        for (auto& category: KiwixApp::instance()->getContentManager()->getCategories()) {
-            if (category != "other" && category != "all") {
-                excludeTags += "_category:"+category;
-            }
-        }
-    }
```

Comment:
Aren't we losing the "other" category ?

Reply:
Is "other" not a real category?

Reply:
Indeed. We had to implement "other" on kiwix-desktop side has we were using a fixed list of categories. But now "other" is a real category.